server/game/cards/03_TWI/events/InPursuit.ts

```diff
@@ -0,0 +1,34 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class InPursuit extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '6401761275',
+            internalName: 'in-pursuit'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Exhaust a friendly unit. If you do, exhaust an enemy unit.',
+            optional: true,
```

Comment:
The ability doesn't say "you may" so it is not optional (if there are no targets it will just do nothing and move the event to discard)

Reply:
removed